.github/workflows/flatpak.yml

```
@@ -0,0 +1,95 @@
+        run: |
+          flatpak build-bundle flatpak-repo dash-evo-tool-linux-${{ matrix.arch }}.flatpak org.dash.DashEvoTool
+
+      - name: Upload Flatpak bundle as artifact
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02  # v4.6.2
+        with:
+          name: dash-evo-tool-linux-${{ matrix.arch }}-flatpak
+          path: dash-evo-tool-linux-${{ matrix.arch }}.flatpak
+
+      - name: Attach bundle to release
+        if: github.event_name == 'release'
+        permissions:
```

actionlint (1.7.10)

[error] unexpected key "permissions" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)